packaging/src/service/brooklyn.service

```diff
@@ -0,0 +1,33 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+#  http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[Unit]
+Description=Apache Brooklyn Service
+Documentation=https://brooklyn.apache.org/documentation/index.html
+
+[Service]
+Type=simple
+WorkingDirectory=/opt/brooklyn/
+Environment="JAVA_OPTS=-Dbrooklyn.location.localhost.address=127.0.0.1 -Dlogback.configurationFile=/etc/brooklyn/logback.xml -Xms256m -Xmx1g -XX:MaxPermSize=256m"
+Environment="CLASSPATH=/opt/brooklyn/conf:/opt/brooklyn/lib/patch/*:/opt/brooklyn/lib/brooklyn/*:/opt/brooklyn/lib/dropins/*"
+ExecStart=/usr/bin/java org.apache.brooklyn.cli.Main launch --localBrooklynProperties /etc/brooklyn/brooklyn.conf --persist auto
```

Comment:
Re `--localBrooklynProperties`, Did you mean to have both /home/brooklyn/.brooklyn/brooklyn.properties and `/etc/brooklyn/brooklyn.conf` as sources for configuration?